src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2338,16 +2255,12 @@ private bool IsEnabledByDefault(int eventNum, bool enable, EventLevel currentLev
             EventLevel eventLevel = (EventLevel)m_eventData[eventNum].Descriptor.Level;
             EventKeywords eventKeywords = unchecked((EventKeywords)((ulong)m_eventData[eventNum].Descriptor.Keywords & (~(SessionMask.All.ToEventKeywords()))));
 
-#if FEATURE_MANAGED_ETW_CHANNELS
```

Comment:
FEATURE_MANAGED_ETW_CHANNELS is defined in coreclr/clr.featuredfines.props and and mono/System.Private.CoreLib/System.Private.CoreLib.csproj.  Are we sure that this isn't propagating?

Reply:
Looks like it's also defined in [https://github.com/dotnet/runtime/blob/215b39abf947da7a40b0cb137eab4bceb24ad3e3/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs](https://github.com/dotnet/runtime/blob/215b39abf947da7a40b0cb137eab4bceb24ad3e3/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs).

Reply:
I think I was mistaken, I am planning to put it this ifdef back